### ISSUE TITLE
Display information and statistic on dashboard

### ISSUE THREAD

**Issue by icerove** [near/near-explorer#350]
### Goals
- Top level stats are prominent
- Search is at least as prominent as top level stats
- Transactions are either removed entirely, or not very prominent (just a few displayed)
- Blocks are not prominent

For specific details see the [attached issues on ZenHub](https://app.zenhub.com/workspaces/app-layer-5e2b7ac4f336abcdc02917e8/issues/near/near-explorer/350)

**Comment by frol** [comment]
I want to sum up all the issues we track here.

The current dashboard has:

* The number of Network Nodes:
  * the first number is the number of nodes that do block production and validation
  * the second number is the total number of nodes that send their telemetry to us that they are online (these are the validation nodes and "read-only" nodes that are maintained by partners, enthusiasts, service providers, etc)
* The most recent Block Height (on average this number gets incremented by 1 every second)
* The total number of transactions through the whole history of the chain
* The number of transactions during the last 24 hours (it is a moving window, but I think it makes more sense to just display a graph with daily stats for the last, let's say, 14 days)
* The total number of accounts
* The list of recent transactions (we consider it is useless already)
* The list of recent blocks (it is not really convenient to use that list as it gets a new item every second, so we either constantly update it or update every 10 seconds replacing the whole list with the new records; I think, the heatmap [github contributions style of graph] or a moving column chart is a better way to present the data)

We have requests to display more stats:

* [#189] Total supply over time [huge NEAR token amounts needs to be reasonably presented] (supply breakdown: Liquid tokens, Staked tokens, Locked tokens, unclaimed / etc tokens)
* [#189] Epoch reward for given epoch [measured in NEAR tokens]
* [#189] Amount of fees burnt over time [measured in NEAR tokens]
* [#122] Gas spent / gas limit over time [measured in Gas/TeraGas/etc] (we will possibly like to add a breakdown across shards)
* [#334] Gas price over time [measured in NEAR/Gas] (there seem to be a separate page for a breakdown, see the issue for details)
* [#189] State size over time [measured in bytes/megabytes/etc] (also across shards)

Overall, we probably want to display all the stats with some track of history, so I guess, we should have all of the "current state" stats backed with a graph of a historical change for that number, for example, a total number of accounts can be displayed as the total number + a graph showing how many accounts were back a few days / months / years ago.

**Comment by icerove** [comment]
Please leave your comments and estimation for the new design @corwinharrell

**Comment by bowenwang1996** [comment]
@frol can you also show the progress of the current epoch (start and projected end height).

**Comment by frol** [comment]
@bowenwang1996 I planned to leave it for the Nodes Dashboard #370. Do you think it is useful for this main dashboard? I guess, the only use I see is for unstake, delegation unstake, and delegation withdraw, but I hope some other tooling will help people with delegation and staking.

**Comment by vgrichina** [comment]
- Daily Active Accounts
- Weekly Active Accounts
- Circulating Supply
- Total Tokens Staked

**Comment by bowenwang1996** [comment]
@frol I see. Yeah I think that works.

**Comment by frol** [comment]
@vgrichina I have added the daily/weekly active accounts to the list (/cc @corwinharrell). The staked tokens were already mentioned in the breakdown of the total supply. As to the "circulating supply", according to @bowenwang1996 comment in Slack ("We need to scan through all accounts and look for those that have lockup deployed and then query smart contract (or decode the state) to see the unlocked amount."), it will take some time to figure it out, but I think it will be under total supply breakdown as well, right?

**Comment by vgrichina** [comment]
>  it will take some time to figure it out, but I think it will be under total supply breakdown as well, right?

makes sense to me

**Comment by icerove** [comment]
> 
* The number of Network Nodes (NOTE: we need them apart as it is confusing now):
  * the first number is the number of nodes that do block production and validation
  * the second number is the total number of nodes that send their telemetry to us that they are online (these are the validation nodes and "read-only" nodes that are maintained by partners, enthusiasts, service providers, etc)

NODE PART: validating node and  online nodes need to separate into two columns @corwinharrell

**Comment by corwinharrell** [comment]
Attaching the design for the dashboard refresh here:

https://www.figma.com/file/tW2iLtA1psHki0D1vKExf1/Dashboard-Refresh?node-id=0%3A1

Some things to note, as I've already reviewed with @icerove:

1. Typography will be changing site-wide. I've provided a typographic scale in addition to the UI design to act as a guide when defining font-sizes and font-weights. This scale will be used for both the explorer and the wallet going forward. The most notable thing here is that heading font-sizes (and others in some cases) all move down the scale 1 step when moving from desktop to mobile (eg, H1 on desktop at 38px becomes 31px on mobile). This is to avoid having unnecessarily large headings on mobile.

2. The search bar will persist in the header for every instance except one: when the user is on desktop and on the homepage of the explorer. At all other times, the search bar will be visible within the site nav.

3. Any questions regarding the design should be left as comments on within the Figma document. Enter comment mode by pressing `C` when viewing the document. I will do my best to respond to any comments left there. I'll also be adding a few small implementation notes within the file and @mention-ing the relevant individuals.

4. Examples of the various states for the new design for html `input`s is also in included in the design.

I will be out this week, but @jakestutzman will be available to assist in providing feedback during QA, as well as offering any help if needed around exporting of assets/images from Figma.

**Comment by ilblackdragon** [comment]
I left a bunch of comments on what is there, but a bigger question is what is not there.
I would expect the main page to show more relevant information, like top apps, circulating supply and more stats.

**Comment by corwinharrell** [comment]
@ilblackdragon thanks for the feedback. I'll be making a few changes in response. To address your question about what is not included: The goal is to do this very iteratively. While we have a big list of things we want to add and include (see above), our primary goals for the most immediate changes were to:

1. Remove Txn and block lists, as they update too quickly to actually be useful for finding a particular txn or block
2. Make search more prominent, since that is the primary method used to find specific txns/accounts/blocks
3. Restructure the dashboard to be more accommodating as we continue to add additional stats / metrics / visualizations
4. Update the design to reflect the newest changes to the brand.

In other words, this next implementation is setting us up to add more to the dashboard in the near future. This epic and the associated issues could have reflected this intent better. If there is anything that is **not** included in this homepage refresh that we think is **critical** for a v0, let me know and we can add it (perhaps circulating supply is one of these?) Otherwise, we will be iterating and adding to the dashboard with follow up work to include more metrics / data / visualizations, that go beyond these initial changes. (There is some prioritization work to be done here so we are not just selecting metrics from a big bucket without a clear plan of how we want to iterate on the explorer)

cc @kcole16

**Comment by icerove** [comment]
@SkidanovAlex
